src/view-slot.js

```diff
@@ -46,6 +46,20 @@ export class ViewSlot {
   }
 
   /**
+   *   Runs the animator against the first animatable element found within the view's fragment
+   *   @param  view       The view to use when searching for the element
+   *   @param  direction  The animation direction enter|leave
+   *   @returns null or a Promise
+   */
+  animateView(view:View, direction:String<enter|leave> = 'enter'):Promise<any> {
```

Comment:
Joel, that isn't valid type syntax. It should just be `string`. If you can fix that, I think we can get this in today.